### DISCUSSION TITLE
Different model for Det and Rec possible?

### DISCUSSION THREAD

**Discussion by yhpark-ckline** [PaddlePaddle/PaddleOCR#14321] [Q&A]
Hi, 
I'm currently using fine tuned rec models but now i see that some of the texts are not even detected.
So I trained a new detection model but I wonder if I can use the fine tuned detection model AND the recognition model at the same time. see below:

**Comment by GreatV** [comment]
Yes, you can use different models for text detection and recognition in PaddleOCR simultaneously. You can specify separate paths for your fine-tuned detection and recognition models when initializing the `PaddleOCR` class. Here's an example based on your query:

```python
from paddleocr import PaddleOCR

ocr_model = PaddleOCR(
    use_angle_cls=True, 
    det_model_dir='path_to_your_fine_tuned_detection_model', 
    rec_model_dir='path_to_your_fine_tuned_recognition_model', 
    lang='your_language_code', 
    show_log=False
)
```

In this setup, you can replace `'path_to_your_fine_tuned_detection_model'` and `'path_to_your_fine_tuned_recognition_model'` with the actual paths to your respective models. This allows you to leverage the strengths of both your custom detection and recognition models effectively. Make sure that the paths and language codes are set correctly to ensure the models load as expected.

---
*Response generated by 🤖 [feifei-bot](https://github.com/GreatV/feifei) | gpt-4o*
